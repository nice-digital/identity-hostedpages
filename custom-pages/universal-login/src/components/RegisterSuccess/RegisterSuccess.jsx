import { Link } from "react-router-dom";
import { Alert } from "@nice-digital/nds-alert";

export const RegisterSuccess = () => {
	return (
		<div>
			<h2 className="mt--0">Thank you</h2>
			<p className="lead">
				We've sent you an email with an activation link. Click on the link to
				verify your details and start using your account.
			</p>
			<Alert type="caution">
				Check your spam folder if you do not receive the activation email.
			</Alert>
			<p>
				<Link data-qa-sel="registerSuccess-link-to-login" to="/">
					Return to sign in
				</Link>
			</p>
		</div>
	);
};
