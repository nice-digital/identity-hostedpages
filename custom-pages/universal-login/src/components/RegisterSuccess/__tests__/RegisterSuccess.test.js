
import { render } from "@testing-library/react";
import { MemoryRouter } from "react-router";
import { RegisterSuccess } from "../RegisterSuccess";

jest.mock("../../../helpers/isLoginPage", () => true);

describe("RegisterSuccess components", () => {
	it("should render RegisterSuccess correctly", () => {
		const { container } = render(
			<MemoryRouter future={{ v7_relativeSplatPath: true, v7_startTransition: true }}>
				<RegisterSuccess />
			</MemoryRouter>);
		expect(container).toMatchSnapshot();
	});
});
