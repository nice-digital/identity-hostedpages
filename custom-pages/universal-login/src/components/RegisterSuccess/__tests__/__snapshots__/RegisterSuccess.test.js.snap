// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RegisterSuccess components should render RegisterSuccess correctly 1`] = `
<div>
  <div>
    <h2
      class="mt--0"
    >
      Thank you
    </h2>
    <p
      class="lead"
    >
      We've sent you an email with an activation link. Click on the link to verify your details and start using your account.
    </p>
    <div
      class="alert alert--caution"
      data-component="alert--caution"
    >
      Check your spam folder if you do not receive the activation email.
    </div>
    <p>
      <a
        data-qa-sel="registerSuccess-link-to-login"
        href="/"
      >
        Return to sign in
      </a>
    </p>
  </div>
</div>
`;
